docs/src/site/_config.yml

```diff
@@ -0,0 +1,5 @@
+name: http4s.org
+markdown: redcarpet
+highlighter: pygments
```

Comment:
If you had to install things to make it work, it would be great to get those noted.  I've had a couple jekyll/pygments sites over the years, and I don't remember what I had to install to make it work.